docs/ref/contrib/postgres/fields.txt

```diff
@@ -916,3 +916,44 @@ types.
     .. attribute:: range_type
 
         The psycopg2 range type to use.
+
+Expression support for range boundaries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: RangeBoundary(lower=LOWER_INC, upper=UPPER_EXC)
+
+For use together with
+:class:`django.contrib.postgres.constraints.ExclusionConstraint`, a simple
+utility class is provided that provides an easier way to write the range
+boundaries. The limits are described in the `documentation`_. Internally,
+``RangeBoundary`` uses :class:`~django.db.models.Value`.
+
+.. _documentation: https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY
+
+String representations of the data types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: RangeOperators()
+
+PostgreSQL provides a set of operators that can be used together with the range
+data types. The operators are described in detail in the documentation. This
+class is only meant as a convenient method to avoid typos in the models::
+
+    class RangeOperators:
```

Comment:
Following on from the complexity bit, we also have to document these constant names rather than just pointing at the list in the PostgreSQL documentation.

Reply:
There's previous feedback from @Ian-Foote about this. Not sure what's the best solution, both to the documentation issue and whether to introduce such a helper class or not.